controller/clusters.go

```diff
@@ -39,3 +44,20 @@ func (c *ClustersController) Show(ctx *app.ShowClustersContext) error {
 	}
 	return nil
 }
+
+func (c *ClustersController) LinkExistingIdentitiesToCluster(ctx *app.LinkExistingIdentitiesToClusterClustersContext) error {
+	// currently using online registration as we already have it atleast for prod-preview, so we can use it directly.
+	if !token.IsSpecificServiceAccount(ctx, token.OnlineRegistration) {
```

Comment:
Let’s use some dedicated temporal service account. “migration” for example. We don’t have add any credentials for that SA. We will manually generate a token for that account. Just hardcode its name in the controller.